packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -48,120 +51,38 @@ const TestProviders = (props: {
  * Render your component, which will be wrapped with all the necessary Provider components
  *
  * @param children
+ * @param props - additional props to pass to providers in TestProvider component
  * @param options - additional options for testing-library's render function
- * @param providerProps - additional props to pass to providers in TestProvider component
  */
 export const renderWithProviders = (
     children: React.ReactElement,
-    options?: Omit<RenderOptions, 'wrapper'>,
-    providerProps?: {
-        commerceApiProvider?: Partial<CommerceApiProviderProps>
-    }
+    props?: TestProviderProps,
+    options?: Omit<RenderOptions, 'wrapper'>
 ): void => {
     render(children, {
         // eslint-disable-next-line react/display-name
-        wrapper: () => <TestProviders {...providerProps}>{children}</TestProviders>,
+        wrapper: ({children}: {children?: React.ReactNode}) => (
+            <TestProviders {...props}>{children}</TestProviders>
+        ),
         ...options
     })
 }
 
-type NockBackOptions = {
-    directory: string
-    mode?: nock.BackMode
-}
-
-export const mockAuthCalls = () => {
```

Comment:
Big change in this PR is, we change the way we mock authentication.

New way to mock authentication is to mock the `Auth` class module.